README.md

```diff
@@ -86,6 +86,7 @@ See also [Static Site Generators](#static-site-generators) and  [Content Managem
   * [Dotclear](http://dotclear.org/) - Take control over your blog! ([Source code](https://hg.dotclear.org/dotclear)) `GPLv2` `PHP`
   * [Ghost](http://ghost.org/) - Just a blogging platform ([Source code](https://github.com/TryGhost/Ghost)) `MIT` `Nodejs`
   * [htmly](http://www.htmly.com/) - Databaseless Blogging Platform (Flat-File Blog) ([Source code]((https://github.com/danpros/htmly)), [Demo](http://www.htmly.com/demo/)) `GPLv2` `PHP`
+  * [Jekyll] (http://jekyllrb.com/) - Simple, blog-aware, static sites. ([Source code](https://github.com/jekyll/jekyll) `MIT` `Ruby`
```

Comment:
We don't list static site generators, please see https://github.com/Kickball/awesome-selfhosted/issues/72, https://github.com/Kickball/awesome-selfhosted/issues/66, https://github.com/Kickball/awesome-selfhosted/issues/41. Can you remove this entry?